app/views/admin/content_assets/new.html.erb

```diff
@@ -1,5 +1,5 @@
+<%= link_to 'Back', admin_content_assets_path, class: 'govuk-back-link' %>
```

Comment:
Why has this button changed position on the page?

Reply:
Regarding some styles changed/elements being moved around, it's just part of the work I did with refactoring the code, I didn't add anything new or change the functionality, they were just some things that I noticed that weren't done properly in the CMS.
For example, with one of the back buttons, I noticed that some page didn't have one where they should (or it was in the wrong position), or the button had the wrong class and so wasn't presenting with the correct styling properly.